src/main/java/org/czekalski/petapiintegration/controllers/AuthorizationController.java

```diff
@@ -1,61 +1,160 @@
 package org.czekalski.petapiintegration.controllers;
 
 
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.czekalski.petapiintegration.apiClient.v1.dto.AnimalDto;
+import org.czekalski.petapiintegration.apiClient.v1.dto.AnimalsListDto;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.security.oauth2.client.OAuth2RestTemplate;
-import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.io.IOException;
+
 @RestController
 public class AuthorizationController {
 
-
-
     @Value("${messages.base-uri}")
     private String messagesBaseUri;
 
     @Autowired
-    private OAuth2RestTemplate messagingClientClientCredsRestTemplate;
+    private OAuth2RestTemplate auth2RestTemplatee;
+
+    @GetMapping(value = "/")
+    public String getHello() {
 
 
+
+        return "hello" ;
+    }
+
     @GetMapping(value = "/authorize", params = "grant_type=client_credentials")
     public String client_credentials_grant(Model model) {
 
         Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
         System.out.println("AUTHENTICATION");
         System.out.println(authentication);
-        String messages = this.messagingClientClientCredsRestTemplate.getForObject(this.messagesBaseUri, String.class);
+        String messages = this.auth2RestTemplatee.getForObject(this.messagesBaseUri, String.class);
         System.out.println(messages);
 
 
         return messages ;
     }
 
 
-
-  /*  @GetMapping("/")
-    public String root() {
-        return "redirect:/index";
+    private static AnimalDto toAnimalDto(String jsonData) throws IOException {
+        ObjectMapper om = new ObjectMapper();
```

Comment:
Lepiej żebyś skorzystał z objectMappera, który jest w polu klasy niż tworzył go za każdym razem. A najlepiej, żeby był on przekazywany do konstrukora. Jeżeli nie istnieje jeszcze w kontekście springa, to po prostu zrób go jako nowego beana. Możesz skorzystać np. z `Jackson2ObjectMapperBuilder` przy tworzeniu takiego beana

Reply:
Ok, będę pamiętał.